.vscode/settings.json

```diff
@@ -7,8 +7,12 @@
     "**/CVS": true,
     "**/.DS_Store": true,
     ".nyc_output": true,
-    "coverage": true,
-    "packages/**/lib": true,
-    "packages/**/lib-test": true
+    "coverage": true
+  },
+  "search.exclude": {
+    "**/node_modules": true,
+    "**/bower_components": true,
+    "**/lib": true,
+    "**/lib-test": true
```

Comment:
That shouldn't bother me. It's also nice for tracing unmapped stack traces (tsc's output is super nice and legible, unlike babel...).